File: sample/src/main/java/ru/terrakok/cicerone/sample/java/AppTabContainerFragment.java
package ru.terrakok.cicerone.sample.java;

import ru.terrakok.cicerone.sample.R;

public abstract class AppTabContainerFragment extends AbsTabContainerFragment {

    @Override
    protected int tabContainerId() {
        return R.id.ftc_container;
    }

    @Override
    protected int tabLayoutId() {
        return R.layout.fragment_tab_container;
    }
}

File: sample/src/main/java/ru/terrakok/cicerone/sample/kotlin/AppTabContainerFragment.kt
package ru.terrakok.cicerone.sample.kotlin

import ru.terrakok.cicerone.sample.R

abstract class AppTabContainerFragment : AbsTabContainerFragment() {

    override val tabContainerId: Int = R.id.ftc_container
    override val tabLayoutId: Int = R.layout.fragment_tab_container
}